cachito/web/status.py

```diff
@@ -25,7 +26,7 @@
     "*": {DATABASE, RABBITMQ},
 }
 
-no_retry_session = requests.Session()
+retry_session = get_requests_session()
```

Comment:
We should keep using the `no_retry_session`. The purpose of the status endpoint is to check the status at a given moment in time, retrying on failures is not desirable.